<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qconf_MainWindow</class>
 <widget class="QMainWindow" name="qconf_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qconf_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Navigate</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeView" name="treeNavigate">
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Search</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineSearch">
           <property name="placeholderText">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="listSearch"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="btnBack">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPosition">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeView" name="treeAll">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="InfoViewWidget" name="textBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionHide_disabled_subtrees"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Name"/>
    <addaction name="actionShow_Range"/>
    <addaction name="actionShow_Data_Value"/>
    <addaction name="separator"/>
    <addaction name="actionShow_normal_Options"/>
    <addaction name="actionShow_all_Options"/>
    <addaction name="actionShow_prompt_Options"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionIntroduction"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOption"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionShow_Name">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Name</string>
   </property>
  </action>
  <action name="actionShow_Range">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Range</string>
   </property>
  </action>
  <action name="actionShow_Data_Value">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Data Value</string>
   </property>
  </action>
  <action name="actionShow_normal_Options">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show normal Options</string>
   </property>
  </action>
  <action name="actionShow_all_Options">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show all Options</string>
   </property>
  </action>
  <action name="actionShow_prompt_Options">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show prompt Options</string>
   </property>
  </action>
  <action name="actionIntroduction">
   <property name="icon">
    <iconset theme="help-faq"/>
   </property>
   <property name="text">
    <string>Introduction</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCollapse">
   <property name="icon">
    <iconset theme="list-remove"/>
   </property>
   <property name="text">
    <string>Collapse</string>
   </property>
   <property name="toolTip">
    <string>Collapse the whole tree</string>
   </property>
  </action>
  <action name="actionExpand">
   <property name="icon">
    <iconset theme="list-add"/>
   </property>
   <property name="text">
    <string>Expand</string>
   </property>
   <property name="toolTip">
    <string>Expand the whole tree</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset theme="edit-undo"/>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionSingleView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SingleView</string>
   </property>
  </action>
  <action name="actionSplitView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SplitView</string>
   </property>
  </action>
  <action name="actionFullView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FullView</string>
   </property>
  </action>
  <action name="actionHide_disabled_subtrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide disabled subtrees</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InfoViewWidget</class>
   <extends>QTextBrowser</extends>
   <header>infoViewWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
